### PR TITLE
Add QuakeJS

### PR DESCRIPTION
**Pull request by begleysm**
Added QuakeJS to Games

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -835,6 +835,7 @@ _Games, game servers and control panels._
 - [node-virtual-gamepads](https://github.com/jehervy/node-virtual-gamepads) - Turn your smartphone into a game controller, keyboard, or touchpad for a remote Linux OS machine. `MIT` `Nodejs/CoffeScript`
 - [Posio](https://github.com/abrenaut/posio) - Geography multiplayer game. `MIT` `Python`
 - [piqueserver](https://github.com/piqueserver/piqueserver) - Server for openspades, the first-person shooter in a destructible voxel world. ([Clients](https://github.com/yvt/openspades)) `GPL-3.0` `Python/C++`
+- [QuakeJS](https://github.com/begleysm/quakejs) - QuakeJS is a port of ioquake3 to JavaScript that can be played in a browser. `MIT` `Nodejs/Javascript`
```
Comment (n8225) [inline README.md:838]: Please change `` `Nodejs/Javascript` to `Nodejs` ``

Comment (begleysm) [reply README.md]: Fixed

**Comment by nodiscc** [comment]
@begleysm can you resend a new pull request with QuakeJS (on top of the current master). There are merge conflicts, it might be easier to reopen a new PR, I won't have time to resolve conflicts before a while.

**Comment by begleysm** [comment]
Closing.  Please reference new PR @ #2203 per @nodiscc request.

**Pull request closed without merging**